src/develop/lightroom.c

```diff
@@ -192,18 +192,18 @@ typedef struct dt_iop_bilat_params_t
 #define LRDT_COLORIN_VERSION 1
 #define DT_IOP_COLOR_ICC_LEN 100
 
-typedef enum dt_iop_color_intent_t
+typedef enum dtlr_iop_color_intent_t
```

Comment:
Why did you rename them? It would make more sense to move the intent definition from the two places (librhtoom.c + color.h) to a single place like common/colorspaces.h and use it everywhere.